pkg/network/state.go

```diff
@@ -601,7 +593,6 @@ func (ns *networkState) RemoveConnections(keys []string) {
 		s += " [%d DNS pid collisions]"
 		s += " [%d time sync collisions]"
 		log.Warnf(s,
```

Comment:
How do you feel about changing this to debug?

Reply:
Debug definitely makes more sense.

Reply:
Actually, I think the intent here is to warn the user when one of these rare conditions occurs, so keeping it as warn makes sense to me.